witwidget/notebook/jupyter/wit.py

```diff
@@ -64,6 +73,13 @@ def __init__(self, config_builder, height=1000, delay_rendering=False):
       the render method is called or the WitWidget object is directly evaluated
       in a notebook cell.
     """
+    self.transfer_block = False
+    self.examples_generator = None
+    self.inferences_generator = None
+    # TODO(wit-dev) This should depend on the example size targeting less than
+    # 10MB per batch to avoid websocket issues.
+    self.batch_size = 1000
```

Comment:
This may be a value I set while being extra conservative FYI.

Reply:
thx, made 10k to match other impls for now